src/components/request/useSettledTableData.tsx

```diff
@@ -12,12 +17,29 @@ function useSettledTableData(
   exploreDisputeTx: string | undefined,
   proposedPrice: oracle.types.ethers.BigNumber | undefined,
   chainId: number,
-  parsedIdentifier: string | undefined
+  parsedIdentifier: string | undefined,
+  request: ActiveRequestParams | ActiveRequestParamsByTransaction | undefined
 ) {
   const [rows, setRows] = useState<IRow[]>([]);
   const [headerCells] = useState<ICell[]>(hc);
   const cid: ChainId = chainId ? chainId : 1;
   useEffect(() => {
+    let formattedIdentifier = parsedIdentifier;
+    try {
+      if (formattedIdentifier === "YES_OR_NO_QUERY" && request) {
```

Comment:
we have a formatting utils file, can you add the formatter as a function there, since this might also be reused in the request table